bin/alluxio

```diff
@@ -147,9 +147,16 @@ function copyDir {
     exit 1
   fi
 
+  ABS_PATH_COMMAND="readlink -f "$1""
```

Comment:
does this work with double quotes inside of double quotes? Perhaps we could leave out the `$1` part and call `${ABS_PATH_COMMAND} $1` on line 159

Reply:
Yes, Both approaches work but I think yours is clearer so I changed it